<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionReferenceGUIClass</class>
 <widget class="QMainWindow" name="GestionReferenceGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des références</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="quitter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterOuvrage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un ouvrage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterJournal">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un journal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimerReference">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer une référence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="affichierBibliographie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher la bibliographie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitter</sender>
   <signal>clicked()</signal>
   <receiver>GestionReferenceGUIClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
